test/subgraph-tests/accounts.test.ts

```diff
@@ -1,5 +1,5 @@
 import { getParifiSdkInstanceForTesting } from '..';
-import { TEST_USER_ID1, TEST_USER_ID2, TEST_USER_ID3 } from '../common/constants';
+import { TEST_USER_ID1 as snxAccountId, TEST_USER_ID2, TEST_USER_ID3 } from '../common/constants';
```

Comment:
Incorrect assumption. SNX account id is a bigint. The ID of the subgraph entity vs. the ID of the snxAccount both are different. Please check subgraph schema and reach out to me for explanation